src/problems/basic_problems.jl

```diff
@@ -32,8 +32,8 @@ Optionally, an initial guess ``u₀`` can be supplied which is used for iterativ
 methods.
 
 ```julia
-LinearProblem{isinplace}(A,x,p=NullParameters();u0=nothing,kwargs...)
-LinearProblem(f::AbstractDiffEqOperator,u0,p=NullParameters();u0=nothing,kwargs...)
+LinearProblem(A,b;u0=nothing,kwargs...)
+LinearProblem{isinplace}(f::AbstractSciMLOperator,b,p=NullParameters();u0=nothing,kwargs...)
```

Comment:
this isn't true yet

Reply:
The surrounding text all mention SciMLOperator, not DiffEqOperator.